### DISCUSSION TITLE
v2ray是否支持Windows的TFO？

### DISCUSSION THREAD

**Discussion by CCCAUCHY** [v2fly/v2ray-core#1583] [Q&A] [answered]
v2ray是否支持Windows的TFO（TCP快速打开）？

**Comment by AkinoKaede** [comment]
支持

**Comment by ghost** [comment]
Actually, TFO not useful in this time.

**Comment by database64128** [comment, marked as answer]
On Windows, V2Ray can create TCP listeners with TFO enabled. Outgoing TCP connections won't be able to take advantage of TFO because you have to call `CONNECTEX()` with specific parameters. I implemented TFO transparently for most platforms in [`database64128/tfo-go`](https://github.com/database64128/tfo-go). It's possible to make changes to V2Ray to use `tfo-go` to fully support TFO.
